packages/features/ee/workflows/lib/reminders/emailReminderManager.ts

```diff
@@ -288,7 +288,7 @@ export const scheduleEmailReminder = async (args: scheduleEmailReminderArgs) =>
     // Can only schedule at least 60 minutes and at most 72 hours in advance
     if (
       currentDate.isBefore(scheduledDate.subtract(1, "hour")) &&
-      !scheduledDate.isAfter(currentDate.add(72, "hour"))
+      !scheduledDate.isAfter(currentDate.add(2, "hour"))
```

Comment:
2 hours should be enough, we are running the corn job to schedule unscheduled emails every 15 minutes